<launch>
  <include file="$(find ros_study)/launch/localisation.launch" />
  <include file="$(find ros_study)/launch/move_base.launch" />
  <include file="$(find ros_study)/launch/map_server.launch" />
  <node name="vehicle_emu" pkg="ros_study" type="vehicle_emu.py" clear_params="true">
    <param name="publish_tf" value="false"/>
    <param name="publish_odom" value="false"/>
    <param name="gps_err" value="1"/>
    <param name="odom_left_err" value="-0.06"/>
    <param name="odom_right_err" value="-0.04"/>
    <remap from="twist" to="odom/twist"/>
    <remap from="fix" to="gps/fix"/>
    <remap from="imu" to="imu/data"/>
  </node>
</launch>
